assets/js/nps.js

```diff
@@ -0,0 +1,41 @@
+function fNpsApi() {
+    // var searchTerm = document.querySelector('#searchTerm').value;
+
+    var searchTerm = 'CA';
+    var sNpsApi = 'https://developer.nps.gov/api/v1/campgrounds?' +
+        'stateCode=' +
+        searchTerm +
+        '&limit=500' +
+        '&api_key=nNVtVCtPxYIdqgXcnAhfGjtCq9cW1SUGJ6AnV68u'
+    // Create a variable to hold the value of rating
+    // var rating = document.querySelector('#rating').value;
+    //  https://developer.nps.gov/api/v1/parks?parkCode=acad&api_key=nNVtVCtPxYIdqgXcnAhfGjtCq9cW1SUGJ6AnV68u
+    fetch(sNpsApi)
+        .then(function (response) {
+            return response.json();
+        })
+        .then(function (response) {
+            console.log(response);
+            aNPS[0].markerName = response.data[0].name;
+            aNPS[0].latitude = response.data[0].latitude;
```

Comment:
Are you selecting Latitude and Longitude seperately because its easier to input into google maps? I don't really think it would matter if we did it this way or used 'latLong' I was just curious about your thought process here!